src/styles/global.css

```
@@ -46,6 +46,31 @@
   }
 }
 
+@layer components {
+  .story-hint-underline {
+    --story-hint-underline: var(--underline-dashed);
+    padding-bottom: 5px;
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)